SensusService/Probes/PollingProbe.cs

```diff
@@ -208,7 +237,32 @@ protected override void InternalStart()
 
                     }, GetType().FullName + " Poll", TimeSpan.FromMinutes(_pollingTimeoutMinutes), userNotificationMessage);
 
+#if __IOS__
+                if (_significantChangePoll)
+                {
+                    _locationManager.RequestAlwaysAuthorization();
+                    _locationManager.DistanceFilter = 5.0;
+                    _locationManager.PausesLocationUpdatesAutomatically = true;
+                    _locationManager.AllowsBackgroundLocationUpdates = true;
+
+                    if (CLLocationManager.LocationServicesEnabled)
+                    {
+                        _locationManager.StartMonitoringSignificantLocationChanges();
+                    }
+                    else
+                    {
+                        SensusServiceHelper.Get().Logger.Log("Location services not enabled; please enable in Settings.", LoggingLevel.Normal, GetType());
+                    }
+
+                    _locationManager.LocationsUpdated += (o, e) => SensusServiceHelper.Get().ScheduleOneTimeCallback(callback, 0);
```

Comment:
You initialize `_locationManager` in the constructor, but then add the event handler here. This will result in multiple event handlers being bound to the location manager, and all will be called. Add the event handler to `_locationManager` in the constructor. Will also need to move `callback` there.